README.md

```diff
@@ -1,2 +1,160 @@
+  </a>
+  
+  <a href="https://bundlephobia.com/result?p=simplify-redux-app">
+    <img src="https://badgen.net/bundlephobia/minzip/simplify-redux-app">
+  </a>
+</p>
+
+Simplify redux app (SRA) is the modern and lightweight (less than 2 kb) toolkit that will simplify your work with redux. SRA allows you to forget all pain with actions, constants, reducers, moreover, it is fully compatible with existing code.
+
+The library is actively used and developed, so if you have any questions welcome! 👋 
+
+
+## 📦 Installation
+
+To use SRA in your project, install it via npm:
+
+```
+npm i simplify-redux-app
+```
+or by yarn:
+```
+yarn add simplify-redux-app
+```
+
+then create module
+
+```jsx
+import { simplifyBuilder } from 'simplify-redux-app';
+
+// create builder that will handle all works
+const builder = simplifyBuilder(state, existReducers);
+
+// create reducer for that will be used by redux
+const reducer = builder.getReducers()
+```
+
+more cool example can be found below
+
+## ✨ Features
+SRA allows you to create actions that can be used to dispatch data to your store. Here is an example of these actions:
+
+```jsx
+import { simplifyBuilder } from 'simplify-redux-app';
+
+export interface ITodoState {
+  items: string[];
+}
+
+export const todoState: ITodoState = {
+  items: ['test'],
+};
+
+// init builder from the library
+const builder = simplifyBuilder(todoState, {});
+
+// example how to create simple redux action
+const addTodoItem = builder.createReduxAction((test: string) => ({
+  name: 'ADD_TODO_ITEM',
+  updater: (state) => ({
+    ...state,
+    items: [...state.items, test],
+  }),
+}));
+
+// example how to send request a server  
+const loadTodoItemById = builder.createServerAction((id: string) => ({
+  name: 'LOAD_TODO_ITEM_BY_ID',
+  url: 'https://jsonplaceholder.typicode.com/todos/' + id,
+  method: httpMethod.get,
+  updater: (state, payload: APITodoItemResponse) => ({
+    ...state,
+    items: [...state.items, payload.title],
+  }),
+}));
+
+// create reducer for redux
+export const rootReducer = combineReducers({
+  todo: builder.getReducers(),
+});
+
+// create store
+export const store = createStore(rootReducer);
+```
+
+for the simple redux store update you can use `createReduxAction`, for example `addTodoItem` will create promise
+```typescript
+const addTodoItem: (test: string) => Promise<SimpleAction<ITodoState, [test: string]>>
+```
+
+and simply by dispatching it your react component `dispatch(addTodoItem('test'))` your redux store will be updated, that's it!
+
+in the case when you need to send the request to the server via HTTP and put the result into redux, you can use `createServerAction`. For example `loadTodoItemById` will will create promise:
+
+```typescript
+const loadTodoItemById: (id: string) => Promise<SimpleAction<ITodoState, APITodoItemResponse>>
+```
+
+and like previously by dispatching the function in the to react component your store will be updated!
```

Comment:
and like previously by dispatching the function in the react component your store will be updated!